### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by Blisolda** [aclap-dev/video-downloadhelper#746] [Help and Q&A]
### Your question

Hi,

This is my first time posting here.

A download from [this site](https://opto.sic.pt/vod/julia-or-07-02-2024/c66adec9-555c-4173-a011-7cefe9704dc4) has failed a few times. Today I tried using the updated version, and it failed again. It usually goes all the way up to 100% and then it fails. 
I followed the suggestion to post here, so here I am.

Here are the bug details:

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://opto.sic.pt/vod/julia-or-07-02-2024/c66adec9-555c-4173-a011-7cefe9704dc4",
  "error": {
    "error": "coapp_failure",
    "details": "Error: Native port disconnected",
    "id": "coapp_failure_9ea8bc70-a0b7-4b3a-b66b-2aa81045c02c",
    "downloadable_id": "downloadable_689153945393733"
  }
}

Thanks in advance!

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: pt-PT
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"b48c845a218d4563"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "C:\\Users\\Utilizador\\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 7
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You could teach a lot of other users how to make a proper problem report.  Thank you for yours.

Sadly, I can't investigate this.  The URL in you page_url tag takes me to a page that nags me for a login.  I'm not going to register for this site.  Oh well . . .

**Reply by Blisolda** [reply]
Thank you for your reply.

I don't know why the site will ask you to login, I never had to do it. Maybe because you're outside Portugal, could be that...
